src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -87,14 +89,17 @@ private static SymbolLookup makeWindowsLookup() {
                     libLookup(libs -> libs.load(jdkLibraryPath("syslookup")));
 
             int numSymbols = WindowsFallbackSymbols.values().length;
-            MemorySegment funcs = MemorySegment.ofAddress(fallbackLibLookup.find("funcs").orElseThrow().address(),
-                ADDRESS.byteSize() * numSymbols, Arena.global());
+            MemorySegment funcs = fallbackLibLookup.find("funcs").orElseThrow()
+                    .reinterpret(ADDRESS.byteSize() * numSymbols);
```

Comment:
I realize now that this might be better expressed as a static final `SequenceLayout` in `WindowsFallbackSymbols`, since the number of symbols doesn't change. WDYT?